build/Dockerfile.allinone

```diff
@@ -0,0 +1,39 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.13.15 AS builder
```

Comment:
will this Dockerfile be used for the MODH opendatahub-operator image?

Reply:
we would keep I think opendatahub-operator image being built from Dockerfile.multistage, but Dockerfile.allinone would replace the build from odh-operator-allinone that is currently our opinionated instance.

Reply:
updated:  we'll move the Dockerfile.multistage to Dockerfile.modh and include this change